src/04-modality-specific-files/05-task-events.md

```diff
@@ -119,13 +119,20 @@ sub-control01/
 Specific fields related to stimulus presentation details SHOULD also be present
 in this data dictionary:
 
-| Column name                      | Description                                                                                                                                                                                                                                  |
+| Field name           | Description                                                                                                                                                                                                                                        |
+|----------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| StimulusPresentation | RECOMMENDED. List of fields related to the software used to present the stimuli during the experiment. Valid fields include `OperatingSystem`, `SoftwareName`, `SoftwareRRID`, `SoftwareVersion` and `Code`. See table below for more information. |
```

Comment:
Instead of `List of fields`, the value for the key `StimulusPresentation` is an [object](https://www.w3schools.com/js/js_json_objects.asp)

Note: in this suggestion I did not correct the table pipes for the linter:

```suggestion
| StimulusPresentation | RECOMMENDED. Object containing key value pairs related to the software used to present the stimuli during the experiment. For example: `OperatingSystem`, `SoftwareName`, `SoftwareRRID`, `SoftwareVersion` and `Code`. See table below for more information. |
```

Reply:
OK I will accept your suggestions and "lint" things after that.